panc/src/test/pan/Functionality/list/list10.pan

```diff
@@ -0,0 +1,10 @@
+# get index greater than size
+#
+# @expect=org.quattor.pan.exceptions.EvaluationException
+#
+object template list10;
+
+"/a" = {
+    t = list(1 ,2);
+    t[-3];
```

Comment:
This seems to be the same test as `list8`. Should it be positive 3?

Reply:
nice catch